acme/utils.go

```diff
@@ -50,7 +50,7 @@ func GetenvOrFile(envVar string) string {
 	fileContents, err := ioutil.ReadFile(fileVarValue)
 
 	if err != nil {
-		fmt.Errorf("Error reading the file at " + fileVarValue + " (defined by " + fileVar + ")", err)
+		fmt.Printf("Error reading the file at " + fileVarValue + " (defined by " + fileVar + ")", err)
```

Comment:
The `Printf` will by lost with the extra arg. Something along that line maybe: `"Error reading %s (by %s). %s", fileVarValue ,fileVar, err`